emails/models.py

```diff
@@ -303,7 +301,7 @@ def emails_replied(self):
     @property
     def joined_before_premium_release(self):
         date_created = self.user.date_joined
-        return date_created < settings.PREMIUM_RELEASE_DATE
+        return date_created < datetime.fromisoformat('2021-10-22 17:00:00+00:00')
```

Comment:
I believe so, since that was the value of this var in `.env-dist`: https://github.com/mozilla/fx-private-relay/pull/2136/files#diff-9b75aa53740e3b569fdd203e0b882eaadadc8c03b5c840e648f37a5b2ca7b622L36

There were a couple of tests that overrode the env var to make sure they preceded or followed the given date, but I went over the (non-removed) ones and verified that the dates were still correct relative to this static date.